packages/access-api/src/utils/ucan-log.js

```diff
@@ -0,0 +1,105 @@
+import pRetry from 'p-retry'
+import * as API from '../bindings.js'
+
+/**
+ *
+ * @param {object} input
+ * @param {URL} input.url
+ * @param {string} [input.auth]
+ */
+export const connect = (input) => new UCANLog(input)
+
+export const debug = () => new UCANLogDebug()
+
+/**
+ * @implements {API.UCANLog}
+ */
+class UCANLog {
+  /**
+   * @param {object} input
+   * @param {URL} input.url
+   * @param {string} [input.auth]
```

Comment:
it could be helpful to include a comment on what kind of string this is. e.g. should it be base64, base64url, what kind of `auth` (http basic?)